packages/account-export/src/lib/helpers/importAccount.ts

```diff
@@ -0,0 +1,62 @@
+import CryptoJS from "crypto-js";
+import generator from "generate-password";
+
+import type { AccountImportData } from "@near-wallet-selector/core";
+
+export const encodeAccountData = (
+  accountsData: Array<AccountImportData>
+): string => encodeURIComponent(JSON.stringify(accountsData));
+
+export const decodeAccountData = (
+  encodedAccountsData: string
+): Array<AccountImportData> =>
+  JSON.parse(decodeURIComponent(encodedAccountsData));
+
+interface EncryptAccountDataProps {
+  accountData: Array<AccountImportData>;
+  secretKey: string;
+}
+
+export const encryptAccountData = ({
+  accountData,
+  secretKey,
+}: EncryptAccountDataProps): CryptoJS.lib.CipherParams => {
+  if (!secretKey) {
+    throw new Error("Secret key is required");
+  }
+  try {
+    return CryptoJS.AES.encrypt(JSON.stringify(accountData), secretKey);
+  } catch {
+    throw new Error("Unable to encrypt account data");
+  }
+};
+
+interface DecryptAccountDataProps {
```

Comment:
Can we move the interfaces to the top of the file just below other imports?

It's been a practice in this project to keep functions and types/interfaces somehow separate.